model/ftn/w3ounfmetamd.ftn

```diff
@@ -1941,320 +2589,341 @@
 !
 !---------- GROUP 3 ----------------
 !
-! IFI=3, IFJ=1
+! IFI=3, IFJ=1, EF
       META => GROUP(3)%FIELD(1)%META
       META(1)%VARNM='ef'
       META(1)%VARNL='wave_elevation_spectrum'
-      IF (NCVARTYPE.LE.3) THEN 
+      META(1)%VARNS='sea_surface_wave_variance_spectral_density'
+      IF (NCVARTYPE.LE.3) THEN
         META(1)%UNITS  = 'log10(m2 s+1E-12)'
-        META(1)%VARNS='base_ten_logarithm_of_power_spectral_density_of_surface_elevation'
+        !META(1)%VARNS='base_ten_logarithm_of_power_spectral_density_of_surface_elevation'
+        META(1)%VARNC='base_ten_logarithm'
         META(1)%FSC     = 0.0004
       ELSE
         META(1)%UNITS  = 'm2 s'
-        META(1)%VARNS='power_spectral_density_of_surface_elevation'
         META(1)%FSC     = 1.
       END IF
       META(1)%ENAME  = '.ef'
       META(1)%VARNG = META(1)%VARNS
       META(1)%VMIN = -12
       META(1)%VMAX = -12.8
```

Comment:
Excuse my interruption, but META(1)%VMAX = -12.8 is an error. I think it was meant to be META(1)%VMAX = 12.8. This max value seems reasonable for netCDF data type short (NCVARTYPE==2), but I don't feel sure this value is large enough for netCDF data type real (NCVARTYPE==4).

Reply:
Because the log10 is taken for netCDF data type short only, there must be different limits `VMIN, VMAX`. For data type real the limits of `ef` can be `META(1)%VMIN = 0; META(1)%VMAX = 1.E12`, and for data type short the variable is `log10(ef+1.E-12)` with limits `META(1)%VMIN = -12; META(1)%VMAX = 12`.

The same issue applies to parameter `P2L`.

Reply:
Hi @CarstenHansen - you are right of course. Originally, I strived to keep the metadata the unchanged for as much of the output as possible, but in this case I managed to get the sign wrong on the VMAX. It would also be better if, as you suggest, there are different limits depending on the NCVARTYPE (due to the log10 scaling). I will make an update in this branch to fix that.

Reply:
Fixed in commit b5a3705